options.c

```diff
@@ -37,6 +37,7 @@ static int verify_region(struct tool_options *options);
 static int verify_and_set_hmr_method(struct tool_options *options);
 static int verify_and_set_hmr_unit(struct tool_options *options);
 static int verify_sg_struct(struct tool_options *options);
+static int verify_and_set_attrsettype(struct tool_options *options);
```

Comment:
I think the function is redundant. You need add  he new parameter to the long_opion : 
 static struct option long_opts[] = {
		{"peer", no_argument, NULL, 'u'}, /* UFS device */
		{"local", no_argument, NULL, 'l'}, /* UFS host*/
                {"static", no_argument, opt->set_type, 1},
2) set_type should be init to 0 in initialized_options function
3) Update the uic help accordingly

Reply:
okay i will update that.